src/App.jsx

```diff
@@ -1,12 +1,49 @@
+
 import './App.css'
-import logo from './assets/logo-white.png'
+import logosmall from './assets/logo-medium.png'
+import {NavLink, Route, Routes} from "react-router-dom";
+import Home from "./pages/Home.jsx";
+import Overview from "./pages/Overview.jsx";
+import NewPost from "./pages/NewPost.jsx";
+import NotFound from "./pages/NotFound.jsx";
+import DetailBlogpost from "./pages/DetailBlogpost.jsx";
 
 function App() {
+    
     return (
-        <div className="page-container">
-            <img src={logo} alt="Company logo"/>
-            <h1>Begin hier met het maken van jouw blog-applicatie!</h1>
-        </div>
+        <>
+            <nav className={"nav-outer-container"}>
```

Comment:
Ik ben nieuwsgierig. :-) Waarom staan de `className`'s tussen accolades?

Reply:
Telkens als je een className selecteert in Webstorm komt hij automatisch met {} en een class is altijd tussen "" alleen vergat ik dan de {} weg te halen. Applicatie blijft het wel doen maar is inderdaad niet netjes! Gelukkig vanaf de andere opdracht al afgeleerd!